templates/files/conf/k8s-addons

```diff
@@ -128,6 +127,23 @@ $KUBECTL -n kube-system delete deploy calico-kube-controllers --ignore-not-found
 # delete legacy nginx daemonset
 $KUBECTL -n kube-system delete daemonset nginx-ingress-controller --ignore-not-found=true
 
+# delete nginx ingress controller installed by terraform <= 5.10
+$KUBECTL -n kube-system delete deploy default-http-backend --ignore-not-found=true
```

Comment:
checked resource names, none of them overlap with the managed app.
If you still think it's insecure we might add a check as we did with the coredns app